provisioning/Samples/device/SymmetricKeyWithOperationalCertificateSample/ProvisioningDeviceClientSample.cs

```diff
@@ -0,0 +1,182 @@
+                // the device, as long as the key has been computed using that value.
+                // Also, the secondary key could be included, but was left out for the simplicity of this sample.
+                using var security = new SecurityProviderSymmetricKey(
+                    _parameters.Id,
+                    _parameters.PrimaryKey,
+                    null);
+
+                using var transportHandler = GetTransportHandler();
+
+                // Pass in your onboarding credentials when creating your provisioning device client instance.
+                // This credential will be used to authenticate your request with the device provisioning service (DPS).
+                ProvisioningDeviceClient provClient = ProvisioningDeviceClient.Create(
+                    _parameters.GlobalDeviceEndpoint,
+                    _parameters.IdScope,
+                    security,
+                    transportHandler);
+
+                _logger.LogInformation($"Initialized for registration Id {security.GetRegistrationID()}.");
+
+                // Pass in your certificate signing request along with the registration request.
+                // DPS will forward your signing request to your linked certificate authority (CA).
+                // The CA will sign and return an operational X509 device identity certificate (aka client certificate) to DPS.
+                // DPS will register the device and operational client certificate thumbprint in IoT Hub and return the certificate to the IoT device.
+                // The IoT device can then use the operational certificate to authenticate with IoT Hub.
+
+                var registrationData = new ProvisioningRegistrationAdditionalData
+                {
+                    OperationalCertificateRequest = certificateSigningRequest,
+                };
+
+                _logger.LogInformation("Registering with the device provisioning service...");
+                DeviceRegistrationResult result = await provClient.RegisterAsync(registrationData);
+
+                _logger.LogInformation($"Registration status: {result.Status}.");
+                if (result.Status != ProvisioningRegistrationStatusType.Assigned)
+                {
+                    _logger.LogError($"Registration status did not assign a hub, so exiting this sample.");
+                    return;
+                }
+
+                _logger.LogInformation($"Device {result.DeviceId} registered to {result.AssignedHub}.");
+
+                if (result.IssuedClientCertificate == null)
+                {
+                    _logger.LogError($"Expected operational certificate was not returned by DPS, so exiting this sample.");
+                    return;
+                }
+
+                // This sample uses openssl to generate the pfx certificate from the issued operational certificate and the previously created ECC P-256 keypair.
```

Comment:
same comment here about "keypair". What is a keypair? I assume you mean key/value pair.

Reply:
I'll update this too.